packages/react/src/PricingOptions/PricingOptions.test.tsx

```diff
@@ -144,4 +144,77 @@ describe('PricingOptions.Item', () => {
 
     expect(testIdsInOrder).toEqual(expectedOrder)
   })
+
+  it('renders the PricingOptions.Heading with a h3 by default', () => {
+    const {getByRole} = render(
+      <PricingOptions.Item>
+        <PricingOptions.Heading>Mock heading</PricingOptions.Heading>
+      </PricingOptions.Item>,
+    )
+
+    const PricingOptionsItemEl = getByRole('heading')
+    expect(PricingOptionsItemEl.tagName).toBe('H3')
```

Comment:
No need to change this, but just fyi you can include the `level` check into the `getByRole`

```suggestion
    expect(getByRole('heading', {level: 3})).toBeInTheDocument()
```